README.md

```diff
@@ -239,9 +249,16 @@ The `depth_qc` attribute values indicate:
 - 0 : no error (does not indicate polygons that only partially overlapped a given reef!)
 - 1 : flags that the reef feature did not overlap any satellite data (value set to 7m)
 - 2 : flags that the minimum value was above sea level (no changes/adjustments made)
+- 3 : flags that depth raster data used for statistic calculation covers less than
+5% of the polygon reef area
```

Comment:
Looking at error code 1, maybe we should flag it with `3` but set the value to 7m as well?